packages/core/contracts/optimistic-assertor/implementation/sovereign-security-manager/SuperbondSovereignSecurityManager.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./BaseSovereignSecurityManager.sol";
+import "../../interfaces/OptimisticAssertorInterface.sol";
+
+contract SuperbondSovereignSecurityManager is BaseSovereignSecurityManager, Ownable {
```

Comment:
it'll be useful to add a block comment above this contract explaining what it does and when it would be useful. it is a bit confusing otherwise, unless you know the point of the superbond system.

Reply:
agree, added